lib/jackson.ts

```diff
@@ -23,27 +33,31 @@ export default async function init() {
     !g.oauthController ||
     !g.adminController ||
     !g.healthCheckController ||
-    !g.logoutController
+    !g.logoutController ||
+    !g.directorySync
   ) {
     const ret = await jackson(env);
     apiController = ret.apiController;
     oauthController = ret.oauthController;
     adminController = ret.adminController;
     logoutController = ret.logoutController;
     healthCheckController = ret.healthCheckController;
+    directorySync = ret.directorySync;
```

Comment:
rename to `directorySyncController` for consistency

Reply:
OK.